<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAnimal</class>
 <widget class="QDialog" name="AddAnimal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="typeLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="typeComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dog</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hamster</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fish</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Snake</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelBreed">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Breed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="breedComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select a Type</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sexLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Sex</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sexComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ageLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ageLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelColour">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Colour</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="colourComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Black</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>White</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Brown</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grey</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sizeLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sizeComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Please Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Small</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Large</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddAnimal</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddAnimal</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
